### ISSUE TITLE
Preflop solver?

### ISSUE THREAD

**Issue by miki151** [bupticybee/TexasSolver#177]
Do I understand correctly that this is only a post-flop solver? Are there any plans to solve preflop scenarios as well? I'm interested in heads up games in particular.

**Comment by bupticybee** [comment]
Yeah, preflop needs mccfr, It's kind of a different algorithm but many technology developed here can be reused for a preflop solver.

But no, I didn't write an preflop solver. However it's not that difficult to write one.
